src/views/ipv/page-multiple-doc-check.njk

```diff
@@ -1,19 +1,32 @@
 {% extends "shared/base.njk" %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% set pageTitleName = 'pages.pageMultipleDocCheck.title' | translate %}
-{% set googleTagManagerPageId = "pageMultipleDocCheck" %}
+{% if context == 'f2f' %}
+  {% set pageText = "pages.pageF2FMultipleDockCheck" %}
+  {% set googleTagManagerPageId = "pageF2FMultipleDockCheck" %}
+{% else %}}
+  {% set pageText = "pages.pageMultipleDocCheck" %}
```

Comment:
Rather than maintaining 2 sets of translations think we should only have one per page as we do atm and define the separate bits of content e.g.
```
"pageMultipleDocCheck": {
        ...,
        "formRadioButtons": {
             ...,
             "otherWayButtonText": "Prove your identity another way",
             "otherWayButtonText_f2f-disabled": "Continue to the service you were trying to use to look for other ways to prove your identity"
        }
}
```